tests/lax_scipy_test.py

```diff
@@ -610,6 +612,13 @@ def test_spectral_dac_svd(self, linear_size, seed, dtype):
     v_eye = jnp.eye(v_unitary_delta.shape[0], dtype=dtype)
     self.assertAllClose(v_unitary_delta, v_eye, atol=eps)
 
+  def test_vq(self):
```

Comment:
I'd feel better if there were more test coverage here... differing numbers of leading dimensions, 1D vs 2D, etc. Perhaps use a parameterized test case?

Reply:
I will try to improve test coverage.